src/sql/workbench/contrib/notebook/browser/notebookExplorer/media/searchview.css

```diff
@@ -0,0 +1,320 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+.search-view .search-widgets-container {
```

Comment:
Are these just copied directly from the vscode search view styles? These need to be scoped down if they're going to live in this file regardless so we don't change/break the existing search view. 

It would also be good to clean up any styles that aren't different than the base search-view styles and only include the differences between the two here.

Reply:
removed this file altogether since all the styles am using can be resued.